ibpc_tester/ibpc_tester/ibpc_tester.py

```diff
@@ -219,15 +220,22 @@ def main(argv=sys.argv):
     # Create list to store results
     results = []
 
+    parts_in_search_phase1_json_path = (
+        "/opt/ros/overlay/install/lib/parts_in_search_phase1.json"
+    )
+    with open(parts_in_search_phase1_json_path, "r") as f:
+        parts_in_search_phase1 = json.load(f)
+
     # Get pose estimates for every image in every scene.
     for scene_id in test_split["scene_ids"]:
         scene_dir = Path(test_split["split_path"]) / "{scene_id:06d}".format(
             scene_id=scene_id
         )
-        scene_gt = load_scene_gt(
-            test_split["scene_gt_rgb_photoneo_tpath"].format(scene_id=scene_id)
-        )
-        for img_id, obj_gts in scene_gt.items():
+        # scene_gt = load_scene_gt(test_split["scene_gt_rgb_photoneo_tpath"].format(scene_id=scene_id))
+
+        # for img_id, obj_gts in scene_gt.items():
+        for img_id, obj_gts in parts_in_search_phase1[f"{scene_id:06d}"].items():
+            img_id = int(img_id)
```

Comment:
If users run with `-e SPLIT_TYPE=val`, the changes here will throw errors. We need to support both sets. One way to do this would be to commit the gt file with the same schema as `scene_gt_rgb_photoneo_tpath` but omit the pose information.